### DISCUSSION TITLE
AXIOM + NUCLEI error >> Could not find template '/root/nuclei-templates': could not find file: stat /root/nuclei-templates: permission denied

### DISCUSSION THREAD

**Discussion by wasp76b** [projectdiscovery/nuclei#1050] [Q&A] [answered]
after the last update of the nuclei engine I'm having problems using the axiom+nuclei, here are the evidences of the error:

![nuclei+axiom_01](https://user-images.githubusercontent.com/75587705/134354334-dae20307-5540-44e7-81b3-bdec74a36734.png)

![nuclei+axiom_02](https://user-images.githubusercontent.com/75587705/134354375-bde2668d-3114-4656-8edb-6dbb4692f39b.png)

the command I have been using is as follows:
axiom-scan 200http -m nuclei -t /root/nuclei-templates -impact low -impact medium -impact high -impact critical -o result

**Comment by ehsandeep** [comment]
@wasp76b you can do the following to achieve the same, as a default all the templates are loaded.

```
nuclei -impact low,medium,high,critical -o result
```

**Comment by wasp76b** [comment]
At first, I already figured that your answer wouldn't solve the problem because I've been using this command for over a year (-impact low -impact medium -impact high -impact critical -o result). but anyway, i did as you suggested (nuclei -impact low,medium,high,critical -o result) and the same problem still persists. the problem is AXIOM + NUCLEI and is not related to the additional parameters of the nuclei.

**Reply by ehsandeep** [reply]
As per the error, looks like you have a permission issue on the running host.

**Reply by wasp76b** [reply]
it's not a permission issue, so I put evidence of the permission of the folder the templates are in to demonstrate that the permission is set correctly (/root/nuclei-templates). See the figure that shows the permission the folder currently has. The permission is correct; but incorrectly the nuclei engine is reporting permission folder issue.

**Comment by wasp76b** [comment]
it's not a permission issue, so I put evidence of the permission of the folder the templates are in to demonstrate that the permission is set correctly (/root/nuclei-templates). See the figure that shows the permission the folder currently has. The permission is correct; but incorrectly the nuclei engine is reporting permission folder issue.

**Comment by 0xtavian** [comment, marked as answer]
@wasp76b Can you post the axiom module youre using for nuclei? Issues that could cause this could be:
1) do you have a copy of the nuclei templates on each remote instance in path `/root/nuclei-templates`? The default path for nuclei templates in axiom is in `/home/op/nuclei-templates`. If youre not using any custom templates you can remove `-t /root/nuclei-templates ` from your command and it will default to `/home/op/nuclei-templates`
2) if you do intend to use templates that are in `/root/nuclei-templates`, you might need to add sudo to axiom module.

**Reply by wasp76b** [reply]
You're correct, the issue was the following path: /home/op/nuclei-templates

![nuclei+axiom_03](https://user-images.githubusercontent.com/75587705/134396221-116857e0-477d-49a7-94eb-79f8407852e6.png)


thanks @0xtavian
